### PR TITLE
[yang] support acl MIRROR_ACTION

### PR DESCRIPTION
**Pull request by wen587**
Fixes https://github.com/Azure/sonic-buildimage/issues/9929
#### Why I did it
ACL doesn't have mirror related action
#### How I did it
Add 'MIRROR_INGRESS_ACTION' and 'MIRROR_EGRESS_ACTION' to sonic-acl.yang.j2
#### How to verify it
Run the YANG model unit tests
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Review by dgsudharsan — CHANGES_REQUESTED** [review]
Please add sample configuration here https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json

**Comment by wen587** [comment]
> Please add sample configuration here https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json

Added.

**Review by dgsudharsan — APPROVED** [review]

**Comment by wen587** [comment]
@ganglyu @qiluo-msft @bingwang-ms  could you help review. Thanks

File: src/sonic-yang-models/yang-templates/sonic-acl.yang.j2 [hunk 1]
```diff
@@ -65,6 +69,18 @@ module sonic-acl {
...
+					type leafref {
+						path "/sms:sonic-mirror-session/sms:MIRROR_SESSION/sms:MIRROR_SESSION_LIST/sms:name";
+					}
+				}
+
+				leaf MIRROR_EGRESS_ACTION {
+					type leafref {
+						path "/sms:sonic-mirror-session/sms:MIRROR_SESSION/sms:MIRROR_SESSION_LIST/sms:name";
+					}
+				}
```
Comment (ganglyu) [inline src/sonic-yang-models/yang-templates/sonic-acl.yang.j2:82]: There are INGRESS_ACTION and EGRESS_ACTION, I think we can check acl rule. For example, acl stage is ingress, MIRROR_EGRESS_ACTION is invalid.

Comment (wen587) [reply src/sonic-yang-models/yang-templates/sonic-acl.yang.j2:82]: https://github.com/Azure/sonic-mgmt/blob/master/tests/everflow/everflow_test_utilities.py#L147-L150
Looks valid action.

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**